fixtures/env.test

```diff
@@ -2,16 +2,22 @@
 
 RUN_ENV=test
 AGENT_HOME=ethereum
+AGENT_REPLICAS_ALL=true
```

Comment:
Is this env var needed? Default behavior can be if no AGENT_REPLICA_{} is specified then we use all replicas defined in the config? So we avoid wrong combinations such as AGENT_REPLICAS_ALL=true and AGENT_REPLICA_0=luckspecialnetwork

Reply:
Default is that if `AGENT_REPLICAS_ALL` is unspecified, code will look for `AGENT_REPLICA_{}`. So not a required field